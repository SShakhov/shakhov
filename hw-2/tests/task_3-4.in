-42 -666.4242 -13.2 -2.718281828459045 1
