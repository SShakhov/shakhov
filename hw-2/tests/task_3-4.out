-723.342481828
